emission/net/api/cfc_webapp.py

```diff
@@ -198,11 +199,12 @@ def getCalPopRoute():
 
 @route("/result/heatmap/pop.route/commute/<selMode>")
 def getCommutePopRoute(selMode):
-  mode = get_mode_db().find_one({'mode_name': selMode})
+  map_mode = {"car" : ecwm.MotionTypes.IN_VEHICLE, "train" : ecwm.MotionTypes.IN_VEHICLE, "walk" : ecwm.MotionTypes.ON_FOOT, "cycling" : ecwm.MotionTypes.BICYCLING, "bus": ecwm.MotionTypes.IN_VEHICLE}
```

Comment:
I don't think that this is the correct mapping. Why use ON_FOOT instead of WALKING? I think that the data contains WALKING almost all the time and very little ON_FOOT. Check it out! This is probably the reason for your error during testing as well.